### PR TITLE
common/TextTable,mgr: standardize on 2 spaces between table columns

### PR DESCRIPTION
**Pull request by liewegas**
```
--- RAW STORAGE ---
CLASS  SIZE     AVAIL   USED     RAW USED  %RAW USED  
ssd    101 GiB  99 GiB  1.0 GiB   2.0 GiB       1.98  
TOTAL  101 GiB  99 GiB  1.0 GiB   2.0 GiB       1.98  
 
--- POOLS ---
POOL  ID  STORED  OBJECTS  USED  %USED  MAX AVAIL  
foo    1     0 B        0   0 B      0     98 GiB  
```

### CODE REVIEW

File: src/common/TextTable.h [hunk 1]
```diff
@@ -82,6 +83,15 @@ class TextTable {
    */
   void set_indent(int i) { indent = i; }
 
+  /**
+   * Set column separation
+   *
+   * @param s String to separate columns
+   */
+  void set_column_separation(const std::string& s) {
```
Comment (tchaikov) [inline src/common/TextTable.h]: where is this method used?

File: src/common/TextTable.h [hunk 2]
```diff
@@ -53,6 +53,7 @@ class TextTable {
   std::vector<TextTableColumn> col;	// column definitions
   unsigned int curcol, currow;		// col, row being inserted into
   unsigned int indent;			// indent width when rendering
+  std::string column_separation = {"  "};
```
Comment (tchaikov) [inline src/common/TextTable.h]:
```suggestion
  constexpr static std::string_view column_separation = {"  "};
```

File: src/common/TextTable.h (hunk 1 above)
Comment (liewegas) [reply src/common/TextTable.h]: no callers

**Comment by liewegas** [comment]
retest this please

**Comment by liewegas** [comment]
retest this please

**Pull request merged**